canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -303,7 +317,8 @@ DropDown.propTypes = {
 		PropTypes.object
 	]), // pass in by redux
 	messageInfo: PropTypes.object, // pass in by redux
-	readOnly: PropTypes.bool
+	readOnly: PropTypes.bool,
+	iconCallBack: PropTypes.func
```

Comment:
How dos this work?  There is nothing passing this callback into this component. 

A couple things are needed
1. Add the callback into https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/common-properties.jsx#L257.  This is an object of callback for common-properties.
2. Once the callback is added you can use the properties controller to access the callback.
3. There needs to be a paramDef updated and the test harness updated to test this callback.  Possibly a toggle that turns it on/off.
4. Unit tests.

Reply:
There is still something missing here with `const { iconCallBack } = this.props;`.  This doesn't get passed in today.

Reply:
I don't think this is actually used.